### ISSUE TITLE
AB:n ruksitejtis jäsentyvät kummasta

### ISSUE THREAD

**Issue by vesal** [TIM-JYU/TIM#3295]
![image](https://user-images.githubusercontent.com/826392/207953361-e505a8ab-d84f-4ad4-b9a3-25ec0b28b2ad.png)

https://tim.jyu.fi/answers/kurssit/tie/ohj2/2022s/demot/DemoC2#taulukko

**Comment by dezhidki** [comment]
Joo, tuo tulee tarpeeksi kapealla näytöllä, koska "Validi" sekä "mukautetut pisteet" -ruksi ovat samassa elementissa:

![image](https://user-images.githubusercontent.com/5202606/208042116-552f3797-a97a-4be5-860b-dd7631540f7f.png)

Ne voisi kapeilla näytöillä laittaa ehkä allekkain, esim mallia

![image](https://user-images.githubusercontent.com/5202606/208042442-5705563f-f423-407e-a437-6355031a88ae.png)

**Comment by vesal** [comment]
mutta onko tuossa myös turha justify?

**Comment by sijualle** [comment]
> mutta onko tuossa myös turha justify?

Varmaan johtuu tuosta flexin tavasta tasoittaa nuo samalle riville, mutta nuo divit tuolla välissä tekee tuosta tasoittamisesta epätasaista.

Ainakin alkuun kannattaa nuo divit heittää pois niin asettuu nätisti, ja sitten laittaa tarvittaessa jollain logiikalla allekkain

https://timdevs01-3.it.jyu.fi/teacher/abflow (kaikki yhdellä rivillä)
https://timdevs01-3.it.jyu.fi/teacher/abflow-noflex (automaattinen rivitys)

**Comment by vesal** [comment]
> https://timdevs01-3.it.jyu.fi/teacher/abflow (kaikki yhdellä rivillä)
> https://timdevs01-3.it.jyu.fi/teacher/abflow-noflex (automaattinen rivitys)
>

Minusta melkein tuo alempi on parempi.  Molemat vievät kaksi riviä.
Ja alempi toimii vaikka näyttöä kaventaa.
Ainoa pieni miinus on ymmärtää kumpaan tekstii ruksi kuuluu.
